website/source/docs/auth/ldap.html.md

```diff
@@ -257,3 +257,528 @@ default, foobar, zoobar
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">userdn</span>
+        <span class="param-flags">optional</span>
+        Base DN under which to perform user search. Example:
+        `ou=Users,dc=example,dc=com`
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">userattr</span>
+        <span class="param-flags">optional</span>
+        Attribute on user attribute object matching the username passed when
+        authenticating. Examples: `sAMAccountName`, `cn`, `uid`
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">discoverdn</span>
+        <span class="param-flags">optional</span>
+        Use anonymous bind to discover the bind DN of a user. Defaults to
+        `false`.
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">deny_null_bind</span>
+        <span class="param-flags">optional</span>
+        This option prevents users from bypassing authentication when providing
+        an empty password. Defaults to `true`.
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">upndomain</span>
+        <span class="param-flags">optional</span>
+        userPrincipalDomain used to construct the UPN string for the
+        authenticating user. The constructed UPN will appear as
+        `[username]@UPNDomain`. Example: `example.com`, which will cause
+        vault to bind as `username@example.com`.
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">groupfilter</span>
+        <span class="param-flags">optional</span>
+        Go template used when constructing the group membership query. The
+        template can access the following context variables:
+        \[`UserDN`, `Username`\]. The default is `(|(memberUid={{.Username}})(member={{.UserDN}})(uniqueMember={{.UserDN}}))`,
+        which is compatible with several common directory schemas. To support
+        nested group resolution for Active Directory, instead use the following
+        query: `(&(objectClass=group)(member:1.2.840.113556.1.4.1941:={{.UserDN}}))`.
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">groupdn</span>
+        <span class="param-flags">optional</span>
+        LDAP search base to use for group membership search. This can be the
+        root containing either groups or users.
+        Example: `ou=Groups,dc=example,dc=com`
+      </li>
+    </ul>
+    <ul>
+      <li>
+        <span class="param">groupattr</span>
+        <span class="param-flags">optional</span>
+        LDAP attribute to follow on objects returned by `groupfilter` in order
+        to enumerate user group membership. Examples: for groupfilter queries
+        returning _group_ objects, use: `cn`. For queries returning _user_
+        objects, use: `memberOf`. The default is `cn`.
+      </li>
+    </ul>
+  </dd>
+
+  <dt>Returns</dt>
+  <dd>`204` response code.
+  </dd>
+</dl>
+
+
+### /auth/ldap/config
```

Comment:
This line can be removed since the `GET` method below comes under the previous header.